mips/pmap.c

```diff
@@ -347,7 +348,7 @@ void tlb_exception_handler(exc_frame_t *frame) {
       log("Address %08lx not mapped by any active pmap!", orig_vaddr);
       goto fault;
     }
-    if (is_valid(pmap->pde[index])) {
+    if (pmap->pde && is_valid(pmap->pde[index])) {
```

Comment:
I believe **this** is the actual fix for that TLB problem I've been trying to reproduce for some longer time (see #161)!